styles.css

```
@@ -0,0 +1,174 @@
+/*    colors*/
+    :root{
+       --paars:#5722CD;
+       --blauw:#4845E4;
+       --roze:#ED4385;
+       --zwart:#0C0C0C;
+       --lichtpaars:#F2F0FF;
+       --lichtgrijs:#F3F6F9;
+       --grijs:#7A7A7A;
+       /*--backgroundgrad:rgb(75,73,231);*/
+       /* background: linear-gradient(135deg, rgba(75,73,231,1) 60%, rgba(96,44,207,1) 100%);*/
+    }
+/* ------------------------------- 2. Typography ----------------------------------- */
+    @font-face {
+        src: url("assets/fonts/Merriweather-Black.ttf");
+        font-family: Merri;
+        font-weight: normal;
+    }
+    @font-face {
+        src: url("assets/fonts/Merriweather-Bold.ttf");
+        font-family: Merri;
+        font-weight: bold;
+    }
+    @font-face {
+        src: url("assets/fonts/Merriweather-light.ttf");
+        font-family: Merri;
+        font-weight: 200;
+    }
+    @font-face {
+        src: url("assets/fonts/Roboto-Black.ttf");
+        font-family: robo;
+        font-weight: normal;
+    }
+    @font-face {
+        src: url("assets/fonts/Roboto-Bold.ttf");
+        font-family: robo;
+        font-weight: bold;
+    }
+    @font-face {
+        src: url("assets/fonts/Roboto-Thin.ttf");
+        font-family: robo;
+        font-weight: 200;
+    }
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+/*    .test{*/
+/*        background:var(--roze);*/
+/*    }*/
+    button{
+        background-color: var(--roze);
+        font-family: robo, serif;
+        font-weight: 400;
+        color:var(--lichtpaars);
+        border-radius: 40px;
+        border-style: none;
+        padding: 15px 20px;
+        margin: 0px 225px;
+        text-transform: uppercase;
+        min-width: 200px;
+        min-height: 50px;
+        box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+    }
+    a:link {
+        color: var(--paars);
+        background-color: transparent;
+        text-decoration: none;
+    }
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+    header{
+        background: rgb(75,73,231);
+        background: linear-gradient(135deg, rgba(75,73,231,1) 13%, rgba(96,44,207,1) 88%);
+        font-family: Merri, serif;
+        padding: 0px 0px;
+        margin: 0px 0px;
+    }
+    nav{
+        display: flex;
+        flex-direction: row-reverse;
+    }
+    li{
+        display: block;
+        font-weight: 300;
+        font-family: robo, serif;
+        margin: 2px;
+        float:right;
+        color:var(--lichtpaars);
+        padding: 10px 15px;
+    }
+    .intro{
+        color: var(--lichtpaars);
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        margin: 50px 33%;
+        text-align: center;
```

Deze kun je vervangen met de `align-items` zodat niet alleen de tekst, maar ook de andere componenten netjes centreren:
```suggestion
        align-items: center;
```

Anders kreeg ik bij het smaller maken van het venster niet alles netjes in het midden